examples/multiplayer-fps/characters/player.tscn

```diff
@@ -0,0 +1,209 @@
+transform = Transform3D(1, 0, 0, 0, 1, 0, 0, 0, 1, 0, 0.705698, 0)
+
+[node name="Camera3D" type="Camera3D" parent="Head"]
+
+[node name="Nose" type="MeshInstance3D" parent="Head"]
+transform = Transform3D(1, 0, 0, 0, 1, 0, 0, 0, 1, 0, 0, -0.25)
+mesh = SubResource("BoxMesh_qmeb2")
+
+[node name="PlayerFPSWeapon" type="Node3D" parent="Head"]
+script = ExtResource("3_4j0gy")
+
+[node name="AudioStreamPlayer3D" type="AudioStreamPlayer3D" parent="Head/PlayerFPSWeapon"]
+stream = ExtResource("5_14ltm")
+
+[node name="BulletHole" type="Node3D" parent="Head/PlayerFPSWeapon"]
+script = ExtResource("6_f1qm8")
+material = SubResource("StandardMaterial3D_faajq")
+
+[node name="TinyGun" type="MeshInstance3D" parent="Head/PlayerFPSWeapon"]
+transform = Transform3D(0.15, 0, 0, 0, 0.15, 0, 0, 0, 0.15, 0.09, -0.18, -0.3)
+mesh = SubResource("BoxMesh_0vqha")
+
+[node name="BigGun" type="MeshInstance3D" parent="Head"]
+transform = Transform3D(1, 0, 0, 0, 1, 0, 0, 0, 1, 0.439686, -0.406913, -0.200588)
+mesh = SubResource("BoxMesh_wqpqp")
+
+[node name="StateNameLabel3D" type="Label3D" parent="."]
+transform = Transform3D(1, 0, 0, 0, 1, 0, 0, 0, 1, 0, 1.30427, 0)
+pixel_size = 0.002
+billboard = 1
+fixed_size = true
+text = "Idle"
+font_size = 16
+
+[node name="Projection3D" type="Node3D" parent="."]
+transform = Transform3D(1, 0, 0, 0, 1, 0, 0, 0, 1, 0, 1.11014, 0)
+script = ExtResource("6_sv41w")
+projection_size = Vector2(1, 0.1)
+
+[node name="ProgressBar" type="ProgressBar" parent="Projection3D" node_paths=PackedStringArray("health")]
+anchors_preset = 15
+anchor_right = 1.0
+anchor_bottom = 1.0
+grow_horizontal = 2
+grow_vertical = 2
+size_flags_horizontal = 3
+theme_override_styles/background = SubResource("StyleBoxFlat_6sg7v")
+theme_override_styles/fill = SubResource("StyleBoxFlat_kndy7")
+step = 1.0
+value = 100.0
+show_percentage = false
+script = ExtResource("17_ghjct")
+health = NodePath("../../Health")
+
+[node name="CollisionShape3D" type="CollisionShape3D" parent="."]
+shape = SubResource("CapsuleShape3D_6kydi")
+
+[node name="MeshInstance3D" type="MeshInstance3D" parent="."]
+mesh = SubResource("CapsuleMesh_xnks4")
+
+[node name="RollbackSynchronizer" type="Node" parent="." node_paths=PackedStringArray("root")]
+script = ExtResource("1_g0p7o")
+root = NodePath("..")
+state_properties = Array[String]([":transform", ":velocity", ":color", "RewindableStateMachine:state", "Head:transform"])
+input_properties = Array[String](["Input:movement", "Input:jump", "Input:fire", "Input:look_angle"])
+
+[node name="TickInterpolator" type="Node" parent="." node_paths=PackedStringArray("root")]
+script = ExtResource("2_3rc26")
+root = NodePath("..")
+properties = Array[String]([":transform", "Head:transform"])
+
+[node name="StateSynchronizer" type="Node" parent="." node_paths=PackedStringArray("root")]
+script = ExtResource("8_p73id")
+root = NodePath("..")
+properties = Array[String](["Health:current_health", ":teleport_position", ":visible"])
```

Comment:
Fun fact: currently there's no way in Godot to ask a node if it has a property by the given name. So what netfox does is it tries to get the value marked by the property entry, and if it's null, it assumes it's not there.

So syncing `:teleport_position` will yield a warning about it being an invalid property. It will be synced regardless.